_includes/search.html

```diff
@@ -1,23 +1,44 @@
 <!-- HTML elements for search -->
+
+<!-- This is installed from the https://github.com/slashdotdash/jekyll-lunr-js-search Gem -->
+<script src="/js/search.min.js" type="text/javascript" charset="utf-8"></script>
+
 <form id="search-form">
   <label for="search-input">
     <h3>Search</h3>
   </label>
-  <input type="text" id="search-input" placeholder="Search posts..">
+  <input type="text" id="search-input" name="q" placeholder="Search posts...">
```

Comment:
```suggestion
  <input autocomplete="off" type="text" id="search-input" name="q" placeholder="Search posts...">
```

Getting the autocomplete dropdown from the browser was a bit annoying to me. This should disable it.

Reply:
Note that the autocomplete from the browser for this search input should be pretty useless...